jstl/1.2/_index.md

```diff
@@ -0,0 +1,38 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta Standard Tag Library Spec 1.2"
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Standard Tag Library encapsulates as simple tags the core functionality common to many Web applications.
+Jakarta Standard Tag Library has support for common, structural tasks such as iteration and conditionals, tags
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
+* [Jakarta Standard Tag TCK](https://download.eclipse.org/jakartaee/jstl/1.2/jstl-tck-1.2.0.zip)
```

Comment:
"Tag TCK" -> "Tag Library TCK"

The URL should be:
https://download.eclipse.org/jakartaee/tags/1.2/eclipse-tags-tck-1.2.0.zip